vcs/util/path.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+// Rel strips the leading "/" prefix from the path string, effectively turning
+// an absolute path into one relative to the root directory.
+func Rel(path string) string {
+	if len(path) > 0 && path[0] == '/' {
+		return path[1:]
+	}
+	return path
```

Comment:
Additionally, are you sure doing the above is the right thing? What if the input is "/", then the output is a blank string. Would that work?

I know in my vfs adapter that I made as a workaround for #23 I had a special case for "/" inputs and returned "." as output. See [here](https://github.com/shurcooL/go/blob/200c7feca7203b642339097c69bdd4d194b549e1/vfs_util/rooted_vfs.go#L38-L46).